package.json

```diff
@@ -100,6 +100,7 @@
     "jest-cli": "16.0.2",
     "jest-webpack-alias": "3.3.0",
     "jison-loader": "1.0.0",
+    "js-beautify": "1.6.14",
```

Comment:
I agree with what you're saying @Poltergeist. Just from playing around with the jsfiddle, I feel it is sufficient for our needs. Thanks for pointing it out! 👍